README.MD

```diff
@@ -1,13 +1,61 @@
 # SeedQueue
+
 ## Description
-SeedQueue is a mod that is meant to replace multi-instancing. Instead of having multiple Minecrafts generating worlds open at the same time, it does it all in just one Minecraft instance as it has a built-in wall screen that can be configured through the SpeedrunAPI menu and further customized through resource packs. This greatly improves performance, especially for lower end hardware, and is also aimed to make speedrunning more accessible.
-## Dependencies
-To use [SeedQueue](https://github.com/KingContaria/seedqueue/releases/latest) you will also need Atum and SpeedrunAPI, you can get them from [here](https://mods.tildejustin.dev/).
-## Latest Release
-Get the latest release of SeedQueue from [here](https://github.com/KingContaria/seedqueue/releases/latest).
-## Tutorial to get started with SeedQueue
-If you are new and want to get started with SeedQueue watch this [tutorial](https://www.youtube.com/watch?v=fGu2MYZxh_c).
-## SeedQueue Wiki
-You can learn more about the config settings for SeedQueue and get guidance for customization of your Wall Screen on the [SeedQueue Wiki](https://github.com/KingContaria/seedqueue/wiki).
-## Discord Server
-If you are looking forward to getting in touch with the SeedQueue Community, join the SeedQueue discord server by clicking [here](https://discord.gg/9P6PJkHCdU).
+
+SeedQueue is a mod for Minecraft Speedrunning that enables quick resetting of worlds by loading multiple worlds at one time. Designed to replace multi-instancing, SeedQueue is able to achieve this with only one instance of Minecraft.
+
+### Key Features
+
+- Built-in "Wall" screen
+- Highly configurable with in game options and resource packs
+- Improved performance, particularly on lower-end hardware
+- Easy to setup
+
+## Installation
+
+> [!IMPORTANT]
+> SeedQueue is only compatible with Fabric Loader version 16.0 and later.
+
+### Prerequisites
+
+1. **Fabric Installation:**  
+    Ensure [Fabric](https://fabricmc.net/use/installer/) is installed. [Prism](https://prismlauncher.org/) and [MultiMC](https://multimc.org/) are easy tools for installing Fabric.
```

Comment:
Do we want to directly recommend prism and multiMC or just link to fabric and leave it at that?

Reply:
Linking to those is definitely good, however I'd list MultiMC first since it's the most widely used in MCSR and change the "and" to "or" so people don't think they need both.